### PR TITLE
'Dogecoin has no Utility' Article

### PR DESCRIPTION
**Pull request by RichieJohn**
Super helpful outline provided by MishaBoar that I've made a few tweaks to. Please amend wherever you see fit.

### CODE REVIEW

File: content/dogepedia/faq/dogecoin-has-no-utility.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ date = "2021-10-22"
...
+Dogecoin was first meant to be a joke - and it served, and will serve, this purpose very well. It almost immediately became a means to tip and encourage others to go and achieve something: the Jamaican bobsled team, the water wells in Kenya, the Dogecoin car, and many more crazy initiatives the community supported over the years. As a tipping currency, thanks to its low fees and speed of transactions in comparison to competing chains for a long part of its existence, Dogecoin has never stopped working for its intended use. 
 
+Over the years, it became possible to use Dogecoin and spend it in online stores. The past 2 years have seen an explosion in the adoption, with Dogecoin payment integrations provided by third parties like BitPay and Coinbase and several providers providing prepaid cards supporting Dogecoin as a funding source. Shops can also integrate directly with the Dogecoin blockchain — and the Dogecoin Foundation — alongside the ecosystem of developers, is planning to facilitate this further by creating libraries that will streamline and standardise implementations.
```
Comment (MishaBoar) [inline content/dogepedia/faq/dogecoin-has-no-utility.md:12]:
```suggestion
Over the years, it became possible to use Dogecoin and spend it in online stores. The past 2 years have seen an explosion in the adoption, with Dogecoin payment integrations provided by third parties like BitPay and Coinbase and several providers providing prepaid cards supporting Dogecoin as a funding source. Shops can also integrate directly with the Dogecoin blockchain. The Dogecoin Foundation, alongside the ecosystem of developers, is planning to facilitate this further by creating libraries that will streamline and standardise implementations. 
```

Comment (MishaBoar) [inline content/dogepedia/faq/dogecoin-has-no-utility.md:12]:
```suggestion
Over the years, it became possible to use Dogecoin and spend it in online stores. The past 2 years have seen an explosion in the adoption, with Dogecoin payment integrations provided by third parties like BitPay and Coinbase and several providers providing prepaid cards supporting Dogecoin as a funding source. Shops can also integrate directly with the Dogecoin blockchain. The Dogecoin Foundation, alongside the ecosystem of developers, is planning to facilitate this further by creating libraries that will streamline and standardise implementations.
```

Comment (MishaBoar) [reply content/dogepedia/faq/dogecoin-has-no-utility.md:12]: Just edited for clarity in the end.

**Review by MishaBoar — CHANGES_REQUESTED** [review]
Great, just added a tiny suggestion for clarity.

File: content/dogepedia/faq/dogecoin-has-no-utility.md (hunk 1 above)
Comment (RichieJohn) [reply content/dogepedia/faq/dogecoin-has-no-utility.md:12]: Much better. Thanks!

**Review by MishaBoar — APPROVED** [review]
Looks like I had missed your edits @RichieJohn, did not get any notification. Merging this!

**Pull request merged**